docs/docs/contributing/workflow.md

```diff
@@ -57,11 +57,17 @@ $ sbt
 To test a specific test tests/x/y.scala (for example tests/pos/t210.scala):
 
 ```bash
-> partest-only-no-bootstrap --show-diff --verbose tests/partest-generated/x/y.scala
+> filterTest .*pos/t210.scala
 ```
 
-Currently this will re-run some unit tests and do some preprocessing because of
-the way partest has been set up.
+As you can see, the `filterTest` task takes a regular expression as its
```

Comment:
The `filterTest` task takes a regular expression as its argument. For example, you could run a negative and a positive test with:

```bash
> filterTest (.*pos/t210.scala)|(.*neg/t210.scala)
```